Update Support Policy (#88)
sous-chefs/website: content/docs/best-practices.md

## Platform Support

> We will attempt to support the version of operating systems currently in support except where there is a compelling reason not to.

Reviewer: ```suggestion
We will attempt to support all operating systems releases currently supported by the OS vendor except where there is a compelling reason not to.
```

> Current major Operating System support lies in the following:

Reviewer: ```suggestion
Current major operating system support lies in the following:
```

- Centos
- Ubuntu
- Redhat
- Amazon Linux 2

> - Macos

Reviewer: ```suggestion
- macOS
```

> - Windows (Latest)

Reviewer: ```suggestion
- Windows
```

> Some cookbooks are not designed to support all of these operating systems, to see a full list of supported operating systems please refer to the `kitchen.yml` in the cookbook in question

Reviewer: ```suggestion
Some cookbooks are not designed to support all of these operating systems. For a full list of supported operating systems please refer to the `kitchen.yml` in the cookbook in question
```

  - resource title
  - a table of available properties
  - a set of examples using the resource

> ## Semvar

Reviewer 2: ```suggestion
## Semver
```

> Our cookbooks must follow semvar <semvar.org> this allows users to understand what the increments mean and plan accordingly

Reviewer: ```suggestion
Our cookbooks follow semvar guidelines. See <semvar.org> for an overview. Semvar allows allows cookbook consumers to better understand what version increments mean and plan their testing regiment accordingly
```

> Our cookbooks follow semvar guidelines. See <semvar.org> for an overview. Semvar allows allows cookbook consumers to better understand what version increments mean and plan their testing regiment accordingly

Reviewer 2: ```suggestion
Our cookbooks follow semver guidelines. See [the semver spec](https://semver.org/spec/v2.0.0.html) for an overview. Semver allows allows cookbook consumers to better understand what version increments mean and plan their testing regiment accordingly
```
